internal/base/options.go

```diff
@@ -6,10 +6,12 @@ package base
 
 // SSTable block defaults.
 const (
-	DefaultBlockRestartInterval      = 16
-	DefaultBlockSize                 = 4096
-	DefaultBlockSizeThreshold        = 90
-	SizeClassAwareBlockSizeThreshold = 60
+	DefaultBlockRestartInterval       = 16
+	DefaultBlockSize                  = 4096
+	DefaultBlockSizeThreshold         = 90
+	SizeClassAwareBlockSizeThreshold  = 60
+	DefaultNumDeletionsThreshold      = 100
```

Comment:
I think these can just stay in options and not here; they seem out of place compared to, say, block sizes, especially without a description.